### ISSUE TITLE
on Julia v1.11 non-allocating internal functions are reported as allocating

### ISSUE THREAD

**Issue by nsajko** [JuliaLang/AllocCheck.jl#60]
On nightly Julia, `f_noalias!` is reported as allocating:

```julia
function f_noalias!(x, y)
  Base.mightalias(x, y) && throw("No aliasing allowed!")
  x .= y
end
using AllocCheck
versioninfo()
check_allocs(f_noalias!, Tuple{Vector{Int}, Vector{Int}})
```

Initially I [reported](https://github.com/JuliaLang/julia/issues/52305) this to Julia, however vtjnash says that at least one of the two internal functions that AllocCheck blames for the allocations does not allocate:

> `jl_genericmemory_copyto` doesn't allocate, so you should file this on AllocCheck instead

**Comment by gbaraldi** [comment]
AllocCheck currently has incomplete information on internal functions and it defaults to marking them as allocating for safety.

**Comment by topolarity** [comment]
This was already fixed in https://github.com/JuliaLang/AllocCheck.jl/pull/58

Planning on making a patch release tomorrow which will include this and other fixes.

**Comment by gbaraldi** [comment]
Oh ok!, will close then
